src/libs/decoders/mp3_seek_table.h

```diff
@@ -37,39 +37,21 @@
 
 #include <vector>    // provides: vector
 #include <SDL.h>     // provides: SDL_RWops
-#include "archive.h" // provides: archive
 
 // Ensure we only get the API
 #ifdef DR_MP3_IMPLEMENTATION
 #  undef DR_MP3_IMPLEMENTATION
 #endif
 #include "dr_mp3.h" // provides: drmp3
 
-// Note: this C++ struct must match (in binary-form) the "drmp3_seek_point" struct
-//       defined in dr_mp3.h.  If that changes, then update this to match, along
-//       with adjusting the Serialize() template function that union's the values.
-//
-struct drmp3_seek_point_serial {
-    drmp3_uint64 seekPosInBytes;      // Points to the first byte of an MP3 frame.
-    drmp3_uint64 pcmFrameIndex;       // The index of the PCM frame this seek point targets.
-    drmp3_uint16 mp3FramesToDiscard;  // The number of whole MP3 frames to be discarded before pcmFramesToDiscard.
-    drmp3_uint16 pcmFramesToDiscard;
-    template <class T> void Serialize(T& archive) {
-            archive & seekPosInBytes & pcmFrameIndex & mp3FramesToDiscard & pcmFramesToDiscard;
-    }
-};
-
 // Our private-decoder structure where we hold:
 //   - a pointer to the working dr_mp3 instance
-//   - a template vector of seek_points (the serializeable form)
+//   - a vector of seek_points
 struct mp3_t {
     drmp3* p_dr = nullptr;    // the actual drmp3 instance we open, read, and seek within
-    std::vector<drmp3_seek_point_serial> seek_points_vector = {};
+    std::vector<drmp3_seek_point> seek_points_vector = {};
 };
 
-uint64_t populate_seek_points(struct SDL_RWops* const context,
-                              mp3_t* p_mp3,
-                              const char* seektable_filename,
-                              bool &result);
+uint64_t populate_seek_points(mp3_t* p_mp3, bool &result);
```

Comment:
Could you update the date in the header comment to 2025?

Reply:
Updated